pkg/metaserver/agent/metric/metric.go

```diff
@@ -287,6 +306,23 @@ func (m *MalachiteMetricsFetcher) updateSystemStats() {
 	}
 }
 
+func (m *MalachiteMetricsFetcher) updateQoSClassCgroupData() {
+	qosClassCgroupPath := []string{m.conf.ReclaimRelativeRootCgroupPath, common.CgroupFsRootPathBurstable, common.CgroupFsRootPathBestEffort}
```

Comment:
the name `qosClass` in this file is kind of confusing, it represents both `k8s qos` and `katalyst qos`,   will it be more intuitionistic if we just name the function as `GetCgroupMetric` and `GetQoSClassNumaMetric`, and the parameter as `cgroup` instead of `qosClass`?

Reply:
done